### PR TITLE
Layout update

### PR DESCRIPTION
**Pull request by malcheg**
@Starzu take a look, please.

### CODE REVIEW

**Comment by Starzu** [comment]
@malcheg layout changes looks cool. I've made some general improvements in this branch, could you review them?

File: guide/src/main/scala/io/udash/web/guide/views/rpc/demos/NotificationsDemoComponent.scala [hunk 1]
```diff
@@ -64,12 +66,29 @@ class NotificationsDemoComponent extends Component {
...
+    val registerButton = UdashButton(buttonStyle = ButtonStyle.Primary)(
+      id := "notifications-demo",
+      produce(model.subProp(_.registered))(
+        p => span(if (!p) "Register for notifications" else "Unregister").render
+      )
+    )
+
+    registerButton.listen {
+      case UdashButton.ButtonClickEvent(btn) =>
+        btn.disabled.set(true)
```
Comment (malcheg) [inline guide/src/main/scala/io/udash/web/guide/views/rpc/demos/NotificationsDemoComponent.scala]: unnecessary line, btn is disabling in onButtonClick function.

File: guide/src/main/scala/io/udash/web/guide/views/rpc/demos/GenCodecsDemoComponent.scala [hunk 2]
```diff
@@ -87,11 +88,20 @@ class GenCodecsDemoComponent extends Component with StrictLogging {
...
     import scalacss.ScalatagsCss._
 
+    val loadIdButton = UdashButton(buttonStyle = ButtonStyle.Primary)(
+      id := "gencodec-demo",
+      "Send request"
+    )
+
+    loadIdButton.listen {
+      case UdashButton.ButtonClickEvent(btn) =>
+        btn.disabled.set(true)
```
Comment (malcheg) [inline guide/src/main/scala/io/udash/web/guide/views/rpc/demos/GenCodecsDemoComponent.scala]: unnecessary line, btn is disabling in onButtonClick function.

**Comment by malcheg** [comment]
@starzu looks good, just check my inline comments.

**Pull request merged**